.github/workflows/tests.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           pip install -r requirements-ci.txt
           python setup.py bdist_wheel
           pip install dist/PennyLane*.whl
+          pip install --upgrade --pre -i https://test.pypi.org/simple/ PennyLane-Lightning

```

Comment:
Note: oh the cool advantage of this approach is that we can actually merge this change is nice :100: (If we were pinning Lightning in `requirements.txt`, then that has to be reverted right before releasing PennyLane.)

Just need to take care when merging the release candidate into `master`. When that's done, we'd like to continue pulling the released version of PennyLane-Lightning on the `master` branch, hence this line should not make it into `master`. By the time the final rc to master PR is being made, PL-LIghtning v0.22.0 will have been released already, so this should be good :+1:

Reply:
This PR can be merged :+1: 

On the day of the release, there's going to be a PR opened that merges `v0.22.0-rc0` into `master`. That PR should not contain this line (although by default it will because of how `git` works).